index.html

```diff
@@ -15,9 +15,9 @@
             <a href="#" class="logo">Tshepo45</a>
              
             <ul class ="navMenu" >
-              <li class="nav-item"><a href="portfolio.html"class="nav-link">Portfolio</a></li>
-              <li class="nav-item"><a href="about"class="nav-link">About</a></li>
-              <li class="nav-item"><a href="Contact" class="nav-link" >Contact</a></li>
+              <li class="nav-item"><a href="#portfolio"class="nav-link">Portfolio</a></li>
+              <li class="nav-item"><a href="#about"class="nav-link">About</a></li>
+              <li class="nav-item"><a href="#Contact" class="nav-link" >Contact</a></li>
```

Comment:
- [ ] you should follow the Figma design and try to implement the the right background color with blur and spaces, margin, padding, font-size, font-family in order to match the Figma design 
- Yours 
![Screenshot 2023-03-14 at 11 40 47](https://user-images.githubusercontent.com/61790587/224976732-a7d0f048-f1f6-49a6-a3ab-87a5672cedfd.png)
- Figma 
![Screenshot 2023-03-14 at 11 41 18](https://user-images.githubusercontent.com/61790587/224976777-7239ceb5-5578-448a-ab6b-ce3828cd798a.png)